tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -1289,88 +1289,6 @@ TEST_F(DxilContainerTest, CompileWhenOkThenCheckRDAT) {
         }
       }
       VERIFY_ARE_EQUAL(resTable.Count(), 8U);
-      // This is validation test for DxilRuntimeReflection implemented on
```

Comment:
Did these tests get moved or removed? I don't see them anywhere in the branch, and that's concerning.